core/src/main/resources/filodb-defaults.conf

```diff
@@ -1,8 +1,19 @@
 filodb {
-
   # list of paths to dataset + ingestion config files
   dataset-configs = [ ]
 
+  # As an alternative to dataset-configs, one can inline list each dataset/stream ingestion config
+  # here, which might be more convenient in some cases than listing file paths (eg tests, reuse of shared configs).
+  # This is only used if dataset-configs is empty.
+  # streams = [
+  #   {
+  #     dataset = "prometheus"
+  #     definition { ... }
+  #     sourceconfig { ... }
+  #   },
+  # ]
+  streams = []
```

Comment:
The reason why I want it called streams is because with multi-schema coming, anything with "datasets" in it probably makes less sense.  However, not a big deal, will change.